# Households comparison - 2050

Pairwise comparison of total households at **2050** between two forecasts, by control/target area and grouped by county.

- **A**: Summer 2026 (pipeline)
- **B (baseline)**: Legacy LUVit (RTP26) (spreadsheet)

`% Diff = (A - B) / B * 100`.

In [1]:
from comparison_helpers import compare_hh_target_area, compare_hh_region

FORECAST_A = 'summer_2026'
FORECAST_B = 'legacy_luvit_rtp26'  # baseline (denominator)
YEAR = '2050'

## Region

In [2]:
compare_hh_region(FORECAST_A, FORECAST_B, YEAR)

KeyError: 'summer_2026'

## King County

In [ ]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53033)

,Summer 2026,Legacy LUVit (RTP26),Diff,% Diff
Bellevue,"101,707","100,738",+969,+1.0%
Seattle,"475,447","485,162","-9,715",-2.0%
Auburn,"40,953","40,308",+645,+1.6%
Bothell,"18,458","18,549",-91,-0.5%
Burien,"28,888","28,921",-33,-0.1%
Federal Way,"49,375","48,943",+432,+0.9%
Issaquah,"19,343","20,152",-809,-4.0%
Kent,"58,028","57,964",+64,+0.1%
Kirkland,"53,315","53,247",+68,+0.1%
Redmond,"51,293","50,671",+622,+1.2%


## Pierce County

In [ ]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53053)

,Summer 2026,Legacy LUVit (RTP26),Diff,% Diff
Tacoma,"138,231","149,610","-11,379",-7.6%
Auburn,"3,731","4,125",-394,-9.6%
Lakewood,"36,727","39,509","-2,782",-7.0%
Puyallup,"26,269","27,912","-1,643",-5.9%
University Place,"20,272","22,196","-1,924",-8.7%
DuPont,"6,185","6,345",-160,-2.5%
Fife,"6,346","6,576",-230,-3.5%
Fircrest,"3,733","4,013",-280,-7.0%
"Mid-County, Parkland-Spanaway-Midland, South Hill","77,401","78,078",-677,-0.9%
Sumner,"6,736","7,237",-501,-6.9%


## Snohomish County

In [ ]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53061)

,Summer 2026,Legacy LUVit (RTP26),Diff,% Diff
Everett,"90,837","84,973","+5,864",+6.9%
Bothell,"15,233","14,089","+1,144",+8.1%
Lynnwood,"31,841","29,785","+2,056",+6.9%
Arlington,"16,821","15,138","+1,683",+11.1%
Bothell MUGA,"18,160","17,554",+606,+3.5%
Edmonds,"29,484","27,177","+2,307",+8.5%
Edmonds MUGA,"2,183","2,135",+48,+2.2%
Everett MUGA,"27,928","26,384","+1,544",+5.9%
Larch Way Overlap,"5,114","4,392",+722,+16.4%
Lynnwood MUGA,"26,642","24,939","+1,703",+6.8%


## Kitsap County

In [ ]:
compare_hh_target_area(FORECAST_A, FORECAST_B, YEAR, 53035)

,Summer 2026,Legacy LUVit (RTP26),Diff,% Diff
Bremerton,"27,034","28,926","-1,892",-6.5%
Bremerton UGA,"5,424","5,910",-486,-8.2%
Silverdale,"13,577","13,914",-337,-2.4%
Bainbridge Island,"12,726","13,257",-531,-4.0%
Kingston UGA,"2,878","3,207",-329,-10.3%
Port Orchard,"11,374","11,535",-161,-1.4%
Port Orchard UGA,"7,495","7,966",-471,-5.9%
Poulsbo,"6,969","7,712",-743,-9.6%
Poulsbo PUTA,646,703,-57,-8.1%
Central Kitsap UGA,"11,951","12,038",-87,-0.7%


## Notes

- `% Diff` is computed as `(A - B) / B * 100`, where **B** (baseline) is the
  second forecast listed at the top of the notebook.
- Cell color: red when `|% Diff| >= 5%`, amber when `>= 2.5%`, green otherwise.
- Target areas present in only one forecast still appear, with `-` for the
  missing forecast's value.